### PR TITLE
Delete test barcodes

### PR DESCRIPTION
**Pull request by squirrelo**
Adds patch to delete the large test barcodes from the system. Also allows verbose patching.

### CODE REVIEW

**Comment by squirrelo** [comment]
Ready for review.

File: amgut/db/patches/0013.sql [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+-- Remove the test barcodes (900000000 and above) so new barcode system works
+DO $do$
+DECLARE
+	bc varchar;
+	sid varchar;
+	agl uuid;
+	agk uuid;
+	pn varchar;
+BEGIN
+-- Wipe out test barcodes taht are already attached to kits and surveys
```
Comment (wasade) [inline amgut/db/patches/0013.sql]: taht -> that

File: scripts/ag [hunk 2]
```diff
@@ -66,6 +66,7 @@ def make(db, force):
     make_settings_table()
     click.echo("Applying patches to database")
     patch_db(verbose=True)
+    migrate_procedures('public', 'ag', verbose=True)
```
Comment (wasade) [inline scripts/ag]: this call always excepts when i run it, does that happen in your env too?

Comment (squirrelo) [inline scripts/ag]: it only excepts for me if you run it outside of make (aka run it on a database that already had it run). That's why I moved it here.

**Comment by squirrelo** [comment]
Ready to go.

**Pull request merged**